sage-ios/SAGE/SAGE/User.swift

```diff
@@ -95,17 +100,21 @@ class User: NSObject, NSCoding {
         for (propertyName, value) in propertyDictionary {
             switch propertyName {
             case UserConstants.kLevel:
-                switch value as! String {
-                case "one_unit": self.level = VolunteerLevel.OneUnit
-                case "two_units": self.level = VolunteerLevel.TwoUnit
-                default: self.level = VolunteerLevel.Default
+                if let val = value as? Int {
+                    switch val {
+                    case 0: self.level = VolunteerLevel.ZeroUnit
+                    case 1: self.level = VolunteerLevel.OneUnit
+                    case 2: self.level = VolunteerLevel.TwoUnit
+                    default: break
+                }
```

Comment:
theyre not supposed to be indented right...all our other switchs aren't

BUT IF THEY ARE OOPS